test/integration/suspend-resume.spec.js

```diff
@@ -80,7 +78,7 @@ describe('QIX Suspend/Resume', () => {
       .then(() => global.deleteApp(app.id))
       .then(() => session.close())
       .catch(error => session.close().then(() => Promise.reject(error)))
-      .then(() => expect(closed.callCount).to.equal(1));
+      .then(() => expect(closed.callCount >= 1).to.equal(true));
```

Comment:
Do we know why there were 2 close events?

Reply:
It's really hard to trigger locally (about 1 in 100 runs for me), but it seems to be related to the `ws` websocket library, there is nothing in the engine responses indicating why, we just seem to get multiple 'close' events on the socket.